firefox_puppeteer/ui/toolbars.py

```diff
@@ -324,16 +324,15 @@ def is_open(self):
     def popup(self):
         """ Provides access to the popup result element.
 
-        :returns: The popup result element.
+        :returns: The popup result element
         """
         return self.marionette.find_element(By.ID,
                                             'PopupAutoCompleteRichResult')
 
     @property
     def results(self):
-        """ Povides access to the container node for autocomplete results.
-
-        :returns: The result container node.
+        """
+        :returns: The autocomplete result container node
```

Comment:
Have you looked at the properties I was referring to in the new tabbar and tab class? It's not mirroring it. I feel we should have a description and return line, and that the description tells about the DOM element.